### DISCUSSION TITLE
Finding location of generated `init.lua` file?

### DISCUSSION THREAD

**Discussion by TRPB** [nix-community/nixvim#2376] [Q&A]
This is a bit of an XY but for debugging I'd like to find the location of the `init.lua` generated by nixvim

I feel like there's a way to do this in neovim itself but I can't find the option.

It's in `/nix/store` under `init.lua` but it's difficult to guess which is the one in use if you've made a number of changes to the config and rebuilt. I'd like to see the file that's being generated to work out what I'm doing wrong in my config.

*Specifically I'm trying to use `plugins.treesitter.highlight.custom_captures` but can't find a good example of it, if I could see the generated file in this case, and others, I'd be able to see what the intent was for the nix object*

**Comment by khaneliman** [comment]
Nixvim-print-init. Otherwise there will be a store location in the neovim wrapper in the nvim command

**Comment by MattSturgeon** [comment]
If you'd like to do this in a nix inspection kinda way, you can access your generated config in the repl and inspect the values under `config.build.*`.

However, if you just want to read the text of your init file, then running `nixvim-print-init` is the correct method.
